apps/web/vibes/soul/primitives/inline-email-form/index.tsx

```diff
@@ -21,10 +22,14 @@ export function InlineEmailForm({
   action,
   submitLabel = 'Submit',
   placeholder = 'Enter your email',
+  successMessage,
+  dismissLabel,
 }: {
   className?: string;
   placeholder?: string;
   submitLabel?: string;
+  successMessage: string;
```

Comment:
Is this the best way to pass in a `success` message for the toast? Open to suggestions.

Wondering if this is also that can be leveraged from conform?

Reply:
Yes, we should be using `submission.reply` from the server and have the payload return a message on success

Reply:
Sorry, not `submission.reply`, but the action payload like we have done in other form actions

Reply:
Updated to return `successMessage`.